modules/ROOT/pages/cloudhub-networking-guide.adoc

```diff
@@ -18,12 +18,17 @@ CloudHub provides a load balancing service for all integrations. You can use the
 
 The CloudHub load balancing service does round robin load distribution across workers, which allows workers to scale linearly as they receive more requests. The load balancing service also provides transparent switchover when an application is upgraded. See xref:managing-applications-on-cloudhub.adoc[Managing Applications on CloudHub].
 
-Each application deployed on CloudHub has a `CNAME` record that refers to the load balancer, for example `myapp.cloudhub.io`. Mule applications deployed on CloudHub must listen on `0.0.0.0` and ports assigned by CloudHub for HTTP and HTTPS. 
+Each application deployed on CloudHub has a `CNAME` record that refers to the load balancer, for example `myapp.cloudhub.io`. Mule applications deployed on CloudHub must listen on `0.0.0.0` and the ports assigned by CloudHub for HTTP and HTTPS. 
```

Comment:
This is incorrect wording in that it sounds like an app can listen on more than one port. Please change wording to "Mule apps deployed in CloudHub must listen on host `0.0.0.0` and to either the HTTP port (8081) or to the HTTPS port (8082)."